.travis.yml

```diff
@@ -1,10 +1,10 @@
 language: node_js
 node_js:
   - "0.10"
+  - "0.12"
+  - "iojs"
```

Comment:
Semi-related, but how will npm work when this gets merged?
The latest Node 0.12 includes npm@2.7.4 ([changelog](https://raw.githubusercontent.com/joyent/node/v0.12.2/ChangeLog))

Does this mean that we'll be downgrading Node 0.12's and iojs's npm _down_ to npm@2.4 (per line 22)?